test/jdk/java/util/Formatter/Basic.java

```diff
@@ -101,6 +90,6 @@ public static void main(String[] args) {
             throw new RuntimeException((fail + pass) + " tests: "
                                        + fail + " failure(s), first", first);
         else
-            out.println("all " + (fail + pass) + " tests passed");
+            System.out.println("all " + (fail + pass) + " tests passed");
```

Comment:
Perhaps use System.out.printf vs println and "fail" should not be needed as all tests passed

Reply:
Thank you Lance, will take a look at this and the rest of the comments

Reply:
Careful.  I suspect that "fail" is the number of tests that are expected to fail.

Reply:
Thanks Iris,
It got hidden in the code snippet, but line 89 is `if (fail != 0)`,
so I do believe fail is redundant in 93.